regions/shapes/tests/test_ellipse.py

```diff
@@ -96,6 +96,9 @@ def test_rotate(self):
         assert_allclose(reg.center.xy, (1, 4))
         assert_allclose(reg.angle.to_value("deg"), 95)
 
+    # TODO: Is this MatplotlibDeprecationWarning something to worry about?
+    @pytest.mark.filterwarnings(r"ignore:The 'rectprops' parameter of "
+                                r"__init__\(\) has been renamed 'props'")
```

Comment:
`rectprops` was deprecated in `matplotlib 3.5` (Nov 16, 2021).  This is fine for now, but I may update this in a separate PR.